docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -172,7 +172,8 @@ Overrides `FLEET_TOKEN_POLICY_NAME` when set.
 [id="env-{type}-fleet-server-service-token"]
 `FLEET_SERVER_SERVICE_TOKEN`
 
-| (string) Service token to use for communication with {es}.
+| (string) Service token to use for communication with {es} and {kib} if <<kibana-fleet-setup>> is enabled.
```

Comment:
```suggestion
| (string) Service token to use for communication with {es} and {kib} if <<env-prepare-kibana-for-fleet,`KIBANA_FLEET_SETUP`>> is enabled.
```
This fixes the link to point to the "[Prepare Kibana for Fleet](https://www.elastic.co/guide/en/fleet/8.7/agent-environment-variables.html#env-prepare-kibana-for-fleet)" anchor, which should fix the failing Docs CI checks.